ledger/metrics/src/main/scala/com/daml/telemetry/package.scala

```diff
@@ -7,5 +7,5 @@ import io.opentelemetry.api.GlobalOpenTelemetry
 import io.opentelemetry.api.trace.Tracer
 
 package object telemetry {
-  val ParticipantTracer: Tracer = GlobalOpenTelemetry.getTracer("participant")
+  val GlobalTracer: Tracer = GlobalOpenTelemetry.getTracer("participant")
```

Comment:
Are we ok to use this "GlobalTracer" across the sdk and "pkv" i.e. get rid of [this one](https://github.com/DACH-NY/daml-on-kv-oem-integration-kit/blob/0e365299528d63dfce8ea3def589ec4bef3f79b4/pkvutils/src/main/scala/com/daml/ledger/telemetry/package.scala#L10)?